src/Missings.jl

```diff
@@ -207,4 +208,261 @@ missing
 """
 passmissing(f) = PassMissing{Core.Typeof(f)}(f)
 
+
+"""
+   skipmissings(args...)
+
+Return a tuple of iterators wrapping each of the iterators in `args`, but
+skipping elements at positions where at least one of the iterators returns `missing`.
```

Comment:
It just occurred to me that it could be useful to mention this keyword:
```suggestion
skipping elements at positions where at least one of the iterators returns `missing`
(listwise deletion of missing values).
```